index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="css/mobile-version.css"/>
+  <link rel="stylesheet" href="css/desktop-version.css"/>
+  <title>Youcode</title>
+</head>
+<body>
+  <!-- Main page section -->
+  <section id="main-page">
+    <header>
+      <nav id="navbar">
+        <a class="my-logo" href="#headline">YouCode</a>
+        <ul class="menu">
+          <li><a class="menuItem" href="#about-course">About</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Program</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">News</a></li>
+        </ul>
+        <ul class="desktop-menu">
+          <li><a class="" href="#about-course">About</a></li>
+          <li><a class="" href="#"style="color: lightgray">Program</a></li>
+          <li><a class="" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="" href="#" style="color: lightgray">News</a></li>
+         </ul>
+        <button class="hamburger">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+      </nav>
+    </header>
+    <main id="headline" class="headline">
+      <h1 class="title">A WEB DEVELOPMENT COURSE TO GET YOU STARTED</h1>
```

Comment:
- [ ] Kindly make sure to use `COCOGOOSE` font family for this title to match the Behance design :recycle: